### ISSUE TITLE
crash when changing theme

### ISSUE THREAD

**Issue by theotheroracle** [florisboard/florisboard#790]
changed theme from light to dark, then floris crashed on open 

#### Environment information
- FlorisBoard Version: 0.3.11-beta02 (37)
- Device: Xiaomi Mi A1
- Android: 10 (cn=Q sdk=29)

#### Features enabled
```kt
smartbar = true
suggestions = true
suggestions_clipboard = true
suggestions_next_word = true
glide = true
clipboard_internal = false
clipboard_history = true
```

#### Attached stacktrace files
<details>
<summary>1619958246992.stacktrace</summary>

```
java.lang.IllegalArgumentException: Cannot round NaN value.
	at kotlin.math.MathKt__MathJVMKt.roundToInt(MathJVM.kt:602)
	at dev.patrickgold.florisboard.ime.text.keyboard.TextKeyboard.layout(TextKeyboard.kt:159)
	at dev.patrickgold.florisboard.ime.text.keyboard.TextKeyboardView.onLayoutInternal(TextKeyboardView.kt:637)
	at dev.patrickgold.florisboard.ime.text.keyboard.TextKeyboardView.notifyStateChanged(TextKeyboardView.kt:179)
	at dev.patrickgold.florisboard.ime.text.keyboard.TextKeyboardView.setComputedKeyboard(TextKeyboardView.kt:168)
	at dev.patrickgold.florisboard.ime.text.smartbar.SmartbarView$onAttachedToWindow$3$1$1.invokeSuspend(SmartbarView.kt:129)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7356)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:940)

```
</details>

**Comment by theotheroracle** [comment]
it happens every time btw

**Comment by patrickgold** [comment]
Thanks for your bug report! This crash is connected to #774, can't though figure out exactly where the NaN suddenly comes from. Will investigate into this.

**Comment by patrickgold** [comment]
Above PR is a potential fix for the NaN crash. If possible could you try out if the [debug APK](https://github.com/florisboard/florisboard/pull/794/checks) if it has been fixed for you? Thanks in advance!

**Comment by theotheroracle** [comment]
yeah that does fix the issue for me

**Comment by patrickgold** [comment]
Thanks for testing it out, I have merged the fix into master, will be available in v0.3.11-beta03 today evening!

**Comment by theotheroracle** [comment]
nice ^•^
